api/app/Http/Controllers/WorkspaceController.php

```diff
@@ -24,12 +25,34 @@ public function index()
 
     public function saveCustomDomain(CustomDomainRequest $request)
     {
+        if (!$request->workspace->is_pro) {
+            return $this->error([
+                'message' => 'You need Pro plan for use this feature.',
+            ], 403);
+        }
+
         $request->workspace->custom_domains = $request->customDomains;
         $request->workspace->save();
 
         return new WorkspaceResource($request->workspace);
     }
 
+    public function saveEmailSettings(EmailSettingsRequest $request)
+    {
+        if (!$request->workspace->is_pro) {
+            return $this->error([
+                'message' => 'You need Pro plan for use this feature.',
```

Comment:
Let's improve the message and be precise, we're talking about custom smtp, not any feature (same for message above)

To write a good message, you can select it, use ctrl + k with cursor and ask it to rewrite it explaining what you want